apparmor.d/groups/lxqt/featherpad

```diff
@@ -0,0 +1,57 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+#include <tunables/global>
```

Comment:
On all profiles, you need to set the abi and use `include` instead of `#include` (this form is deprecated) :
```
abi <abi/3.0>,

include <tunables/global>
```